src/commands/run_aws.yml

```diff
@@ -2,14 +2,15 @@ description: >
   Run a command that requires AWS access
 
 parameters:
-  cmd:
-    type: string
-    description: "command to run"
-  title:
+  cmd_name:
```

Comment:
Title felt weird here since the actual parameter is name. Since for some reason name cannot be directly used, I tried `cmd_name`.